lib/ios/TabBarItemAppearanceCreator.m

```diff
@@ -13,13 +13,25 @@ + (void)setTitleAttributes:(UITabBarItem *)tabItem titleAttributes:(NSDictionary
     tabItem.standardAppearance.stackedLayoutAppearance.normal.titleTextAttributes = titleAttributes;
     tabItem.standardAppearance.compactInlineLayoutAppearance.normal.titleTextAttributes = titleAttributes;
     tabItem.standardAppearance.inlineLayoutAppearance.normal.titleTextAttributes = titleAttributes;
+    if (@available(iOS 14.0, *)) {
+      UIColor *color = [titleAttributes valueForKey:@"NSColor"];
+      if (color) {
+        tabItem.standardAppearance.inlineLayoutAppearance.normal.titleTextAttributes = @{NSForegroundColorAttributeName: color};
+      }
+    }
 }
 
 + (void)setSelectedTitleAttributes:(UITabBarItem *)tabItem selectedTitleAttributes:(NSDictionary *)selectedTitleAttributes {
     [super setSelectedTitleAttributes:tabItem selectedTitleAttributes:selectedTitleAttributes];
     tabItem.standardAppearance.stackedLayoutAppearance.selected.titleTextAttributes = selectedTitleAttributes;
     tabItem.standardAppearance.compactInlineLayoutAppearance.selected.titleTextAttributes = selectedTitleAttributes;
     tabItem.standardAppearance.inlineLayoutAppearance.selected.titleTextAttributes = selectedTitleAttributes;
+    if (@available(iOS 14.0, *)) {
+      UIColor *color = [selectedTitleAttributes valueForKey:@"NSColor"];
```

Comment:
Please replace `@"NSColor"` with the existing constant `NSForegroundColorAttributeName`

Reply:
@danilobuerger ah, good point-- updated. thanks!